subsys/bluetooth/controller/Kconfig

```diff
@@ -512,8 +512,21 @@ config BT_CTLR_ADV_SET
 	help
 	  Maximum supported advertising sets.
 
+config BT_CTLR_ADV_INTERVAL_MAX
+	int "Primary Advertising Interval Maximum"
+	depends on BT_BROADCASTER
+	range 32 16384 if BT_LL_SW_SPLIT
+	range 32 16777215
+	default 16384 if BT_LL_SW_SPLIT
+	default 16777215
+	help
+	  Supported Primary Advertising Interval Maximum value.
+	  Bluetooth Core Specification v5.3 permitted range is 0x000020 to
+	  0xFFFFFF in 0.625 ms units.
```

Comment:
Should we hex `hex` instead of `int` to make it easier to compare with the spec values, e.g. `0x000020` and `0xFFFFFF`? 

Also applies to BT_CTLR_ADV_PERIODIC_INTERVAL_MAX

Reply:
Done